src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -15,11 +15,14 @@
 package codeu.controller;
 
 import codeu.model.data.Conversation;
+import codeu.model.data.Tag;
 import codeu.model.data.User;
 import codeu.model.store.basic.ConversationStore;
 import codeu.model.store.basic.UserStore;
+import com.google.appengine.repackaged.com.google.api.client.util.Lists;
```

Comment:
What is this used for?

Reply:
Good question! It was an import Eclipse did automatically...